### PR TITLE
feat: web-www NextJS migration

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: 
regen-network/regen-registry#1661
regen-network/regen-registry#1642
regen-network/regen-web#1845

Page migrated:
- [/token](https://steady-dodol-4090c8.netlify.app/token) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/2bea83c61895a81333cabe9c646b27ffc90552d0))
- [/faq](https://steady-dodol-4090c8.netlify.app/faq) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/31f442c2e15bd57385f7a2c008717ba170d49175))
- [/media](https://steady-dodol-4090c8.netlify.app/media) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/c1cdbceeab789dd1c0fa8a397c7807e2e66ab712))
- [/press-kit](https://steady-dodol-4090c8.netlify.app/press-kit) ([commit](https://github.com/regen-network/regen-web/commit/f51045e4e3856c546392bfeb67077b5dc3900391))
- [/team](https://steady-dodol-4090c8.netlify.app/team) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/8ff84d5e310c3aa11fb8dc4b155ef4e908fbfc8f))
- [/contact](https://steady-dodol-4090c8.netlify.app/team) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/4dba0db0e5c74111df3ed6b8edc9448bc9ab7c96))
- [/resources](https://steady-dodol-4090c8.netlify.app/resources) ([commit](https://github.com/regen-network/regen-web/pull/1889/commits/a9690b0983e05dc3ce930407a638947707cc1004))

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1. Token page: https://steady-dodol-4090c8.netlify.app/token
2. Faq page: http://localhost:3000/faq/concept

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can check the migrated token page, for potential regressions, here: https://steady-dodol-4090c8.netlify.app/token

**Comment by erikalogie** [comment]
> @erikalogie you can check the migrated token page, for potential regressions, here: https://steady-dodol-4090c8.netlify.app/token

Ok everything looks good except for some reason there is this extra space on mobile: https://monosnap.com/file/YkAo5TNE9p29d7gUiqdYKz2rbJovB5

**Comment by flagrede** [comment]
@erikalogie FAQ page is ready to be tested. I also fix the issue on the token page (it was because of the newsletter form that was overflowing).

File: web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          }}
+        >
+          {altContent?.description ? (
+            altContent.description
+          ) : (
+            <BlockContent content={content?.bodyRaw} />
+          )}
+        </Label>
+        <NewsletterForm
+          apiUri={process.env.GATSBY_API_URI}
```
Comment (blushi) [inline web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx:57]: should we add .env and netlify.toml config for that?
this form is not working as is

Comment (flagrede) [reply web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx:57]: I have to migrate these env variables yes, will do that.

Comment (flagrede) [reply web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx:57]: The env file is already set up and the `netlify.toml` defined in `regen-web` already has everything we need, it's just about renaming these env variables.

Comment (blushi) [reply web-www/components/templates/shared/EmailSubmitSection/EmailSubmitSection.tsx:57]: right, we should get rid of `GATSBY_` prefix which doesn't make sense anymore 😅

**Comment by blushi** [comment]
> @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).

getting 404 on /faq

**Comment by flagrede** [comment]
> > @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).
> 
> getting 404 on /faq

thanks for letting me know, there was an issue with the build, it is fixed now.

**Comment by erikalogie** [comment]
> @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).

Great, everything looks good except the back button doesn't seem to work on mobile

**Comment by erikalogie** [comment]
> @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).

Great, everything looks good except the back button doesn't seem to work on mobile

**Comment by flagrede** [comment]
> > @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).
> 
> Great, everything looks good except the back button doesn't seem to work on mobile

@erikalogie  Thanks I completely missed that back button, this is now fixed. Also the media page is up for review: https://steady-dodol-4090c8.netlify.app/media

**Comment by erikalogie** [comment]
> > > @erikalogie FAQ page is ready to be tested here: https://steady-dodol-4090c8.netlify.app/faq. I also fixed the issue on the token page (it was because of the newsletter form that was overflowing).
> > 
> > 
> > Great, everything looks good except the back button doesn't seem to work on mobile
> 
> @erikalogie Thanks I completely missed that back button, this is now fixed. Also the media page is up for review: https://steady-dodol-4090c8.netlify.app/media

Ok the FAQ looks good now.

On the media page, the triangles for the video icons are too big. Otherwise looks good to me.

**Comment by flagrede** [comment]
@erikalogie [press-kit page](https://steady-dodol-4090c8.netlify.app/press-kit) is ready to test.

**Comment by flagrede** [comment]
@erikalogie you can also check out the team page https://steady-dodol-4090c8.netlify.app/team

**Comment by flagrede** [comment]
@erikalogie the awards section should now be centered and the [resources page](https://steady-dodol-4090c8.netlify.app/resources) (this is the last one) is up for testing.

**Comment by erikalogie** [comment]
> press-kit page

Looks good, but I don't see the community calendar link.

**Comment by flagrede** [comment]
@erikalogie sorry about that, for some reason the Netlify deployment with the calendar change failed. I deployed it again and it's now displayed.

**Review by blushi — APPROVED** [review]
Great work! I went through all the migrated pages and everything seems to work properly (except the newsletter form but this is because of CORS issue...)

Just one thing that should be addressed separately, while testing /contact page, I realized that our current /contact registry-server endpoint responds with an error (but a successful banner is still being displayed...), it was relying on some application in airtable that doesn't seem to exist anymore where the contact info were stored, I don't have access to airtable so will reach out to Sarah Bax and @erikalogie to see what we can do about this

**Comment by erikalogie** [comment]
> @erikalogie sorry about that, for some reason the Netlify deployment with the calendar change failed. I deployed it again and it's now displayed.

@flagrede can you move the community calendar to the last item in the list? Sorry, I should have been more specific in the instructions!

**Comment by erikalogie** [comment]
> Great work! I went through all the migrated pages and everything seems to work properly (except the newsletter form but this is because of CORS issue...)
> 
> Just one thing that should be addressed separately, while testing /contact page, I realized that our current /contact registry-server endpoint responds with an error (but a successful banner is still being displayed...), it was relying on some old `Contact` table in airtable that doesn't seem to exist anymore where the contact info were stored, I don't have access to airtable so will reach out to Sarah Bax and @erikalogie to see what we can do about this

Yeah I don't really know much about why that form was deleted so best to reach out to Sarah Bax, though you can keep me in the loop if I can be of assistance as I do have Airtable access.

**Pull request merged**